src/stubs/migration.stub

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('table_name', table => {
```

Comment:
Also, migrations are not only for creating tables. We should do a tableName check on this template like it's done here. 

https://github.com/tgriesser/knex/blob/master/src/migrate/stub/js.stub#L3-L8

Reply:
Hmm. I've tried this before `d` returns empty object.